### DISCUSSION TITLE
How to do free transform with Tranformer?

### DISCUSSION THREAD

**Discussion by dirkarnez** [konvajs/konva#1523] [Q&A]
![image](https://user-images.githubusercontent.com/26408605/221488767-4b52ab08-8ed4-4ba4-ad57-cb7a5e04ea9e.png)
Is there a way to override the rectangular-based transform behavior to do free transform like this (as in Photoshop), please? Thank you!

**Comment by lavrton** [comment]
No, Konva doesn't support that.
